server/config-docs.ts

```diff
@@ -69,6 +69,7 @@ const validator = ajv.compile({
               properties: {
                 title: { type: "string" },
                 slug: { type: "string" },
+                hideInScopes: { type: ["string", "array"] },
```

Comment:
for the second entry, is it possible to be more specific about the `array` type, as in: `string[]`? not sure with this format. we want to stop the possibility of passing anything else in the array but strings.

Reply:
Yes, you're right. That would be better, of course.